docs/future-work/project-roadmap.md

```diff
@@ -19,15 +19,15 @@ Tasks are categorized in two ways:
 
 Tasks that are done have a check mark next to them and are shaded in green. When a task gets checked off, it will add progress to the corresponding capabilities on the right.
 
-# What the TBP Team @Numenta is Working on
+# What the TBP Team is Working On
```

Comment:
Wouldn't "on" qualify as a short preposition?

Reply:
Not according to Vale.

![image](https://github.com/user-attachments/assets/31d06829-3429-4497-aa71-dfe358c983e1)

Reply:
I think this instance it's a linking verb, which should be capitalized.

Reply:
Ah okay, good to know :)